datemath/helpers.py

```diff
@@ -142,7 +148,7 @@ def parse(expression: str, now: Any = None, tz: str = 'UTC', type: Any = None, r
     rettime = evaluate(math, time, tz, roundDown)
 
     if type:
-        return getattr(rettime, type)
+        return cast(Arrow, getattr(rettime, type))
```

Comment:
In strict mode, mypy will whine about returning `Any` from a typed function.
This can either be cast or type-ignored to accept the "type unsafety".
```suggestion
            return getattr(rettime, type)  # type: ignore[no-any-return]
```